src/main.rs

```diff
@@ -484,17 +511,10 @@ struct UiConfig {
     primary_layer: LayerType,
     secondary_layer: LayerType,
     font: String,
-    icon_theme: String,
+    media_theme: String,
```

Comment:
Shouldn’t here be app_icon_theme. Did you compile the daemon before making the change?

Reply:
good catch. I didnt test it as I need to revert some stuff.